### ISSUE TITLE
Add Config for RPi 2/3

### ISSUE THREAD

**Issue by JavaRockstar** [owenthewizard/rpi-wifi-bridge#1]
Can't get RPI WiFi Bridge Working With Raspberry Pi 3B I am not sure where I am going wrong from what I understand this image will not give a graphical display I am fine with that. The problem I am facing I burn the image Beta v3 to the SD card with Etcher I edit WPASupplicant.conf with the wireless information of the wireless device that will be providing internet. I then edited the hosts file changing the hostname to name-tunnel. After these changes, I insert the SD card back into the RPI 3B the red and green lights flicker I don't have access to the serial console so I am relying on editing the files directly by mounting the SD card in a virtual machine to edit the configurations. 

1. Burnt SD Card image v3 with Etcher in Windows
2.  Mounted SD Card in Ubuntu Virtual Machine Edited Config Files
3. Inserted SD Card into RPI 3B
4. Plugged Ethernet cable into the machine that will get access to the wireless device's internet connection.

Does this software work with all variants of the Raspberry Pi or only the Raspberry Pi W Zero with Ethernet HAT?

I like how this tool is created to be minimal using Buildroot and I've learned quite a bit while trying to get this tool working is there a chance you could help me get this tool fully configured with my RPI 3B also is there any external wireless cards supported in this tool thanks.

I am going to try to burn the image directly with dd instead of Etcher in my Windows machine I do an IP address scan with Angry IP Scanner and I can't see the RPI or that hostname connecting to the network.

Can you confirm I would need RPi0w or would this image also work with RPI3B and other RPI models? thanks

**Comment by owenthewizard** [comment]
Editing the hosts file is not the correct way to change the hostname, you will want to undo my custom `S02hostname` and revert to the default buildroot one. Then you can set it in `/etc/hostname`, I think....

This project can support any Wi-Fi or Ethernet interface supported by Linux but may need additional firmware added to the image. Currently there's only a config for the RPi Zero W included, but it's easy enough to use buildroot's config for the RPi 3 and port over the changes.

If I have time I might whip up an included config for RPi 2/3 sometime soon.

Bottom line, it will work but you'll have to assemble it yourself.

**Comment by owenthewizard** [comment]
Checkout the new release and let me know if it works for you.

**Comment by owenthewizard** [comment]
Closing due to lack of activity, re-open this issue if you have any issues with the new RPi2+ release.
